### ISSUE TITLE
cc in quick reply is filled with the own mail address

### ISSUE THREAD

**Issue by irgendwie** [owncloud/mail#743]
I use a mail account, which is configured to receive mails from different mail addresses.

For example:
`max.mustermann@example.com`
`mustermann.max@example.com`
`m.mustermann@example.com`
`...`

(I think you get the idea)

Let's say I use `max.mustermann@example.com` for the account setup. If I now receive a mail via `m.mustermann@example.com`, `m.mustermann@example.com` is automatically filled in the cc field of the quick reply - Instead of this behavior, the cc field should be empty. :-1:

**Comment by DeepDiver1975** [comment]
This is basically the alias support we want to implement for 0.2

**Comment by jancborchardt** [comment]
No, that’s separate. The bug @irgendwie describes can be fixed by simply not putting the alias in the cc field.

**Comment by jancborchardt** [comment]
Right – we need to know which is an alias for this, related to #122
